great_expectations_provider/operators/great_expectations.py

```diff
@@ -17,97 +17,160 @@
 # under the License.
 #
 
-import logging
+import datetime
+import os
+from typing import Any, Callable, Dict, List, Optional, Union
+import uuid
 
-from airflow.models import BaseOperator
+import airflow
 from airflow.exceptions import AirflowException
+
+if airflow.__version__ > "2.0":
+    from airflow.hooks.base import BaseHook
+else:
+    from airflow.hooks.base_hook import BaseHook
+from airflow.models import BaseOperator
 from airflow.utils.decorators import apply_defaults
 import great_expectations as ge
 from great_expectations.checkpoint import LegacyCheckpoint
-
-log = logging.getLogger(__name__)
```

Comment:
Why remove the logger?

Reply:
The operator logging is done with `self.log.info()`, etc, as is the recommended approach for AF operators, AFAIK.  `self.log` is provided by the `LoggingMixin` super class.  For instance, line 161.